src/fiskaltrust.ifPOS/v1/Models/JournalRequest.cs

```diff
@@ -2,7 +2,11 @@
 using System.Runtime.Serialization;
 
 namespace fiskaltrust.ifPOS.v1
-{
+{   /// <summary>
```

Comment:
This will potentially not be formatted correctly, we should add a newline above the `/// <summary>` and remove the one at the end

Reply:
I applied the suggested changes.